sdk/core/Azure.Core.TestFramework/src/RecordedTestSanitizer.cs

```diff
@@ -5,12 +5,15 @@
 using System.Globalization;
 using System.Linq;
 using System.Text;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
 
 namespace Azure.Core.TestFramework
 {
     public class RecordedTestSanitizer
     {
         public const string SanitizeValue = "Sanitized";
+        public static List<string> JsonPathSanitizers = new List<string>();
```

Comment:
Does it have to be `public` or `protected` is enough?

Reply:
great point. will do